target_postgres/json_schema.py

```diff
@@ -99,7 +108,7 @@ def _helper_simplify(root_schema, child_schema):
         try:
             return _helper_simplify(root_schema, get_ref(root_schema, child_schema['$ref']))
         except RecursionError:
-            raise Exception('Target `$ref` "{}" is recursive'.format(get_ref(root_schema, child_schema['$ref'])))
+            raise Error('Target `$ref` "{}" is recursive'.format(get_ref(root_schema, child_schema['$ref'])))
```

Comment:
Given the name of the repo/project, the word "Target" here may be confusing. Maybe '`$ref` path "{}" is recursive'?